udata/core/dataset/views.py

```diff
@@ -103,6 +104,19 @@ def get_context(self):
         return context
 
 
+@blueprint.route('/r/<uuid:id>', endpoint='resource')
+def resource_redirect(id):
+    '''
+    Redirect to the latest version of a resource given its identifier.
+    '''
+    dataset = Dataset.objects(resources__id=id).first()
+    if dataset:
+        resource = get_by(dataset.resources, 'id', id)
+    else:
+        resource = CommunityResource.objects(id=id).first()
```

Comment:
Because `get()` raise an exception if the object does not exists and I needed to have a `none` value in this case. I don't wanted to  add an extra `try...catch` just to set it to `None`